podcasts/MainTabBarController.swift

```diff
@@ -137,9 +139,24 @@ class MainTabBarController: UITabBarController, NavigationProtocol {
         Settings.shouldShowInitialOnboardingFlow = false
     }
 
+    private func fixTarBarTraitCollectionOnIpadForiOS18() {
+        if #available(iOS 18.0, *),
```

Comment:
I tested and it all works as expected.

However, if we merge this one we'll run into the following issue:

1. This code requires Xcode 16 to build
2. However, we cannot submit final builds to Apple using beta Xcode
3. Which means that we will have to change both the `IMAGE_ID` and comment this code

I've been researching a way to do that by using the `$(XCODE_VERSION_MAJOR)` without any luck so far though.

Reply:
I was able to run and build this code on Xcode 15. All the calls made are already present on iOS 17, the only thing that changed in iOS 18 was the way iOS implements the tab bar so only on iOS 18 we need to do these calls.
I think I can revert the CI configurations changes and will still get the CI green.